### ISSUE TITLE
Enabling the script and moving any window's titlebar to screen edge makes KWin drop to 1 FPS

### ISSUE THREAD

**Issue by IlyaBizyaev** [tcorreabr/Parachute#71]
Somehow the maximize preview animation conflicts with Parachute.

Retried with 0.9.0 after reading about performance fixes, but it's still there.

**Comment by tcorreabr** [comment]
I couldn't reproduce it here. Can you make a short video showing the problem?

**Comment by IlyaBizyaev** [comment]
https://user-images.githubusercontent.com/11175213/102691904-f0b96280-4220-11eb-85ef-a0f6db7e1897.mp4

**Comment by tcorreabr** [comment]
Wow! Definitely not happening here. Have you tried disabling other scripts? Maybe there is some kind of conflict between them.

**Comment by IlyaBizyaev** [comment]
I did, disabling other scripts does not help

**Comment by Kodehawa** [comment]
> Also, it seems moving windows immediately becomes laggier after enabling Parachute...

Yep. Everything is at least twice as laggy with this script now. I love it, but at least it seems like I'm not the only one getting absolutely horrendous lag :)

**Comment by IlyaBizyaev** [comment]
Tested with Parachute 0.9.1 on Plasma 5.21, the issue persists.
